Update Add WAN replication configuration section [MC-2711] (#314)
hazelcast/management-center-docs: docs/modules/clusters/pages/wan-replication.adoc

You can add a temporary WAN replication configuration dynamically to a
cluster. It is useful for having one-off
WAN sync operations. The added configuration is not persistent, so it does not survive a member restart.

> If you configure a data structure to use the new WAN configuration, entries added after configuration change

Reviewer: ```suggestion
If you configure a data structure to use the new WAN configuration, entries added after the configuration change
```

> will be automatically replicated by the new WAN replication configuration.

Reviewer: ```suggestion
are automatically replicated by the new WAN replication configuration.
```
